changelog.d/13647.removal

```diff
@@ -0,0 +1 @@
+Remove the ability to direct TCP replication with workers. Workers now require using Redis.
```

Comment:
```suggestion
Remove the ability to use direct TCP replication with workers. Direct TCP replication was deprecated in Synapse v1.18.0. Workers now require using Redis.
```
I personally think this context is useful enough to have directly in the changelog.